Invalid SearchkitComponent type

The current SearchkitComponent type is incomplete. When trying to assing a SearchkitComponent to a typed function taking a React.ComponentClass as parameter, it fails with the following error: 
```
error TS2345: Argument of type 'typeof RefinementListFilter' is not assignable to parameter of type 'ComponentClass<any>'.
  Types of property 'contextTypes' are incompatible.
    Type '{ searchkit: Requireable<any>; }' is not assignable to type 'ValidationMap<any>'.
      Index signature is missing in type '{ searchkit: Requireable<any>; }'.
```

--- a/src/core/react/SearchkitComponent.ts
+++ b/src/core/react/SearchkitComponent.ts
@@ -21,7 +21,7 @@ export class SearchkitComponent<P extends SearchkitComponentProps,S> extends Rea
   stateListenerUnsubscribe:Function
   translations:Object = {}
 
-  static contextTypes = {
+  static contextTypes: React.ValidationMap<any> = {
 		searchkit:React.PropTypes.instanceOf(SearchkitManager)
 	}
 

